app/src/main/java/com/example/stretchy/features/executetraining/sound/managers/SoundEventConsumer.kt

```diff
@@ -0,0 +1,97 @@
+package com.example.stretchy.features.executetraining.sound.managers
+
+import android.content.Context
+import com.example.stretchy.R
+import com.example.stretchy.features.executetraining.sound.Player
+import com.example.stretchy.features.executetraining.sound.Speaker
+import com.example.stretchy.features.executetraining.sound.data.SoundTrack
+import com.example.stretchy.features.executetraining.sound.data.SoundType
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+class SoundEventConsumer(
```

Comment:
what is the purpose of the class other than having some functions? I mean the class should at least manipulate some state. Its some sort of helper I guess, but I am not sure if this is neccessary? Maybe some other forms of abstracting this logic. ordinary functions/ extension functions / extracting to .kts file etc. I dont feel the need for such a abstraction